src/server.js

```diff
@@ -100,6 +113,14 @@ in ${config.public ? "public" : "private"} mode`);
 
 		manager.autoloadUsers();
 	}
+
+	function shutdown() {
+		log.info("Quitting...");
+		server.close(); // Removes socket file
+		process.exit();
+	}
+
+	process.on("SIGINT", shutdown);
```

Comment:
If the server loses power, the socket file will be deleted next time when it is started up again, if the socket file is in /tmp or /run hierarchies like it probably should. But yes, a proper error message would be nice